File: exs/mundo_1/javascript/002.js
/*
Desafio 002

Problema: Faça um programa que leia o nome de uma pessoa e mostre
          uma mensagem de boas-vindas.

Resolução do problema:
*/

const input = require('readline-sync').question
const print = console.log

let nome = input('Qual seu nome? ')
print(`Prazer em conhecer você, ${nome}`)

File: exs/mundo_1/rust/002.rs
/*
Desafio 002

Problema: Crie um script que leia o nome de uma pessoa e mostre uma mensagem de
          boas-vindas de acordo com o valor digitado
*/

fn main() {
    let mut name = String::new();

    println!("Digite seu nome: ");
    std::io::stdin()
        .read_line(&mut name)
        .expect("Falha ao ler o nome");

    println!("Olá, {name}");
}
